build/webpack.config.content.prod.js

```diff
@@ -0,0 +1,24 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+import webpack from 'webpack';
+import contentConfig from './webpack.config.default.content';
+
+export default {
+  ...contentConfig,
+  plugins: [
+    ...contentConfig.plugins,
+    new webpack.NoErrorsPlugin(),
+    new webpack.DefinePlugin({
+      'process.env': {
+        NODE_ENV: JSON.stringify('production'),
+      },
+    }),
+  ],
+  debug: true,
```

Comment:
It's unclear from webpack's docs what this does and why we'd want it in prod but not dev, can you elaborate?